assets/translations/fr.json

```
@@ -3,5 +3,20 @@
   "foreground_notification": {
     "text": "Cette notification permet à Movna de suivre votre position en arrière plan.",
     "title": "L'application Movna est en cours..."
+  },
+  "permissions": {
+    "grant": "Autoriser",
+    "location": {
+      "notification": {
+        "body": "Cette application a besoin d'accéder à la localisation de l'appareil pour enregistrer une activité.",
```

```suggestion
        "body": "Cette application a besoin d'accéder à la localisation de l'appareil pour enregistrer une activité",
```
Sorry but still incohérent 😝